components/quick_switch_modal/quick_switch_modal.jsx

```diff
@@ -305,7 +312,15 @@ export default class QuickSwitchModal extends React.PureComponent {
                             renderDividers={renderDividers}
                             delayInputUpdate={true}
                             openWhenEmpty={true}
+                            onSuggestionsReceived={this.handleSuggestionsReceived}
                         />
+                        {!this.state.hasSuggestions &&
+                        <NoResultsIndicator
+                            variant='Search'
```

Comment:
can we use the `Search` enum here? unless jsx won't allow it 😭.

If that's the case mind putting a comment here so whoever converts this to TS can use the enum.
`// NoResultsVariant.Search`